docs/release.md

```diff
@@ -0,0 +1,81 @@
+- We need a stable tested version of the load-test tool to run the performance comparisons.
+- The RC-1 version of Mattermost is cut at around T-14[^1].
+
+[^1]: Mattermost release process specifies all the release events (feature freeze, RC-1 branch cut, final build cut...) by using the T-X notation, where X is the number of working days before the release day.
+
+As such, we base the timeline of our own release process on the following goal:
+
+> We aim to have a new release ready on T-15.
+
+There may be times when there are no changes in the load-test tool, and that's fine. In those cases, we can skip the release altogether.
+
+## Process
+
+The goal of this release process is to have a well-tested version where bugs can be surfaced before we use it for the monthly performance comparisons and other analysis. Then, the main part of it will be the testing process, and we will skip any release whose tests did not pass before the testing period ends.
+
+- T-30 (owned by Dev): Judge whether we need a new release: there are significant changes since last version. If there are, cut the release branch and continue with next steps. If there are not, skip the release.
+- From T-29 to T-16 (owned by QA, assisted by Dev): Testing period.
+    - Run a performance comparison test with bounded and unbounded tests, Postgres and MySQL, using the latest stable release of Mattermost as both the base and new builds. This test will pass if and only if:
+        - All four tests finish successfully.
+        - There is no significant difference between the base and new builds (they're the same build, and thus should ideally output the same exact results).
+    - If bugs are found in the load-test tool:
+        - Fix them in master.
+        - Cherry-pick them to the release branch.
+        - Run the test again.
+- T-16 end of day (owned by Dev): judge whether the tests successfully passed and decide if the release will happen the next day.
+- T-15 (owned by Dev): If the tests passed, release a new version. If not, skip the release for this month.
+- From T-14 (owned by Dev): Run the actual performance comparison with the latest stable version of the load-test tool.
+
+### Owners
+
+There are two roles specified in this process:
+
+- Dev: the main owner of the whole process, in charge of cutting the release branch, assisting during the testing period, judging whether the release is ready and releasing the new version.
+- QA: the main owner of the testing period, in charge of running the tests and filing the bugs.
+
+The current people in charge of these roles are:
+- Dev: @agarciamontoro
+- QA: @DHaussermann
+
+In the case that the primary Dev and QA are not available, we define Dev and QA backups:
+- Dev backups: @streamer45, @agnivade, @isacikgoz
+- QA backups: @agarciamontoro, @streamer45, @agnivade, @isacikgoz
+
+
+## Release
+
+This section outlines the technicalities of the release itself, once the new version is tested and approved.
```

Comment:
Maybe we could specify it's the binary release or bundle we are referring to here. Mostly because we are using the word *release* from different levels.

Reply:
Do you mean the ltagent/ltapi vs ltctl? At some point we should release ltctl as well I think, otherwise it'll be confusing.

Reply:
I mean the Github release bundle, whatever that may contain.

Reply:
https://github.com/mattermost/mattermost-load-test-ng/pull/640/commits/9d18f1b7f1f03a463e4e599dad9c8d23229b98d4